distribution/client/src/lib/update.js

```diff
@@ -113,7 +121,7 @@ class Update {
   }
 
   saveUpdateSync(manifest) {
-    logger.info('Saving a new manifest..');
+    logger.info('Saving a new manifest.. into ' + this.updatePath());
```

Comment:
In the future please prefer template strings over string concatenation. If i could find an eslint rule for this I would add that :slightly_frowning_face:

Reply:
yeah, right. Will do.